libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -224,6 +224,11 @@ size_t WiFiClient::write(const uint8_t *buf, size_t size)
     return res;
 }
 
+size_t WiFiClient::write_P(PGM_P buf, size_t size)
+{
+    return write(buf, size);
+}
+
```

Comment:
Ok, scratch that — we wouldn't be able to reuse this on the 8266, as that WiFiClient uses different implementation for _P and non-_P. So it's​ just two methods here, for WiFiClient and WebServer. I'd say just add them, as they will be eliminated if not used.